packages/webapp/src/_app/hooks/utils.ts

```diff
@@ -23,13 +23,38 @@ export const useInterval = (callback: () => void, delay: number | null) => {
 const now = () => new Date();
 const padTime = (time: number) => time.toString().padStart(2, "0");
 
+export const useTimeout = (callback: () => void, delay: number | null) => {
```

Comment:
nit: null -> undefined

Reply:
The `delay` is designed to explicitly be set to `null`, which clears the timeout. I'll add a comment linking to where I got this from (https://usehooks-typescript.com/react-hook/use-timeout) but it follows the same interface as Dan Abramov's `setInterval`, which we also have in our codebase and which also takes a `number | null` for this.